common/djangoapps/course_modes/helpers.py

```diff
@@ -115,4 +116,11 @@ def get_course_final_price(user, sku, course_price):
         user.username,
         price_details.get('total_incl_tax')
     )
-    return price_details.get('total_incl_tax', course_price)
+    result = price_details.get('total_incl_tax', course_price) 
+
+    # As per REV-2260: if zero cents, then only show dollars
+    cents = Decimal(result) % 1
+    if cents == 0: 
+        result = int(result)
+
```

Comment:
Could you simplify this check and lose the extra import by returning `int(result)` if  `int(result) != result`?

Reply:
ooh that's simpler- good idea!

Reply:
oh actually- I think that would make a new problem: this would *also* not match for courses that have cents ($49.99), and would truncate that down to $49. My original solution last week had been to just always return an int, but the enterprise folks wanted to preserve the cents where they exist, so the smallest fix I can make for the bug at hand is just to trim it off for this weird case where it's $49.0.

Reply:
For posterity I had it backwards. I had meant:

```
# Return the integer value only if the cents are 00
if int(result) == result:
  return int(result)
return result
```

Reply:
Confirmed that works- and sooo much cleaner- thank you!